priv/repo/migrations/20220819164826_add_foreign_references_and_delete_behaviors_to_tables.exs

```diff
@@ -0,0 +1,54 @@
+defmodule Ret.Repo.Migrations.AddForeignReferencesAndDeleteBehaviorToTables do
+  use Ecto.Migration
+
+  @tables_missing_references [
```

Comment:
I would drop the `tables_` prefix. It’s confusing from the module attribute.

Reply:
Since it’s three pieces of data, I’d just call them tuples or triples or drop the noun. E.g.`@missing_reference_triples`, `@missing_reference_tuples`, `@missing_reference`

Reply:
It’s just a nit though 😄